package.json

```diff
@@ -6,12 +6,13 @@
   "scripts": {
     "start": "node build/index.js",
     "dev": "nodemon --exec babel-node server/index.js",
-    "test": "nyc mocha server/test/test.js --require @babel/register --exit",
+    "test": "npm run drop && npm run create && nyc mocha server/test/test.js --require @babel/register --exit",
     "build": "rm -rf build && mkdir build && babel -d ./build ./server",
     "heroku-postbuild": "npm run build",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "coveralls": "nyc --reporter=lcov --reporter=text-lcov npm test",
-    "create": "cross-env NODE_ENV=development babel-node server/usingDB/migration/createTable.js"
+    "create": "cross-env NODE_ENV=development babel-node server/usingDB/migration/createTable.js",
```

Comment:
are you using the same database for development and production?
I think you should create separate databases.